handlers.go

```diff
@@ -16,8 +16,22 @@ func GetBeers(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 }
 
 // GetBeer returns a beer from the cellar
-func GetBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	// TODO
+func GetBeer(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	ID, err := strconv.Atoi(ps.ByName("id"))
+	if err != nil {
+		fmt.Fprintln(w, ps.ByName("id"), "is not a valid Beer ID, it must be a number.")
```

Comment:
Maybe better to return a 400 here with the message instead?

So something like:
```
w.Header().Set("Content-Type", "application/json") // this could just be done at the top for all responses
w.WriteHeader(http.StatusBadRequest)
json.NewEncoder(w).Encode(fmt.Sprintf("%s is not a valid Beer ID, it must be a number.", ps.ByName("id")))
```

Reply:
We don't want to set application/json if the response won't be in a JSON format, no?

Reply:
Gotcha